docs/orchestrating-elastic-stack-applications/elasticsearch/volume-claim-templates.asciidoc

```diff
@@ -47,4 +47,4 @@ spec:
           emptyDir: {}
 ----
 
-CAUTION: Using `emptyDir` is not recommended due to the high likelihood of permanent data loss.
+CAUTION: Using `emptyDir` is not recommended as it might lead to permanent data loss.
```

Comment:
Is there a way to make this sound scarier? :smile: 
When I read "might lead to", my feeling is that it is not that dangerous. Is there a nicer way of saying "don't do this unless you absolutely know what you are doing"?